### ISSUE TITLE
Reset command

### ISSUE THREAD

**Issue by metalmatze** [mattes/migrate#319]
Hey!
What happened to the reset command? I want to simply reset the db while developing. It seems that there once was a reset command: #123 

Thanks.

**Comment by jeffreylo** [comment]
What you’re describing seems equivalent to a `drop` and `up`.

**Comment by metalmatze** [comment]
Yes, pretty much running `drop` and then `up`. But I'm always either running:
`migrate -verbose -database postgres://... -path migrations drop`
`migrate -verbose -database postgres://... -path migrations up`
separately, or I use `&&` in between. 
I'd simply prefer to run one command with a `reset` instead. :wink:

**Comment by jeffreylo** [comment]
Then write your own command. That’s what this library provides.
